R/read.any.R

```diff
@@ -58,6 +61,8 @@
 #' # to handle type conversions manually:
 #' read.any('date-time.csv', folder = folder, all_chars = TRUE)
 #'
+require(glue)
```

Comment:
for packages, required packages should be identified in DESCRIPTION, not via `require`.